pages/profiles/index.js

```diff
@@ -91,6 +100,17 @@ function SubTree({ tree, objectType, schemas, collectionTitles = null }) {
             >
               {title}
             </Link>
+            <Tooltip content={schema.description}>
+              <Button
+                onClick={() => {}}
+                label={`View the summary of the ${title} schema`}
+                type="secondary"
+                size="sm"
+                hasIconCircleOnly
+              >
+                <QuestionMarkCircleIcon />
+              </Button>
+            </Tooltip>
```

Comment:
Seems a little painful to have to use a button that doesn’t do anything everywhere we’ll need a tooltip. Can the `Popover.Button` be styled the way we need it? Or just use `<Button>` inside `<Tooltip>` instead of `<Popover.Button>`?